lib/xapi/problem.rb

```diff
@@ -32,6 +32,8 @@ def error_message
         "We don't have problems in language '#{track_id}'"
       elsif unknown_problem?
         "We don't have problem '#{slug}' in '#{track_id}'"
+      elsif !readme?
+        "We don't have readme for '#{slug}' in '#{track_id}'"
```

Comment:
We can assume that if we have the problem, we have the README. We have a script that runs on Travis CI with every commit: https://github.com/exercism/x-api/blob/master/bin/verify-metadata